central/image/service/service_impl.go

```diff
@@ -303,6 +307,20 @@ func (s *serviceImpl) ScanImage(ctx context.Context, request *v1.ScanImageReques
 	if request.GetForce() {
 		enrichmentCtx.FetchOpt = enricher.UseImageNamesRefetchCachedValues
 	}
+
+	if request.GetCluster() != "" {
```

Comment:
As mentioned above, this part is also subject to refactoring and creating a util for it.

Reply:
I moved more of the 'duplicate' code into the util method - believe that's inline with the ask now: https://github.com/stackrox/stackrox/pull/7738/commits/177eeddbf27caa0c7f50eb031220982b05b04fe3